pkg/stanza/decode/decode.go

```diff
@@ -1,7 +1,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package decoder // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/decoder"
+package decode // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/decode"
```

Comment:
This must be a breaking change from Go API perspective. Should we keep deprecated aliases for one version or mark it as a breaking change in the CHANGELOG-API?

Reply:
Good call, new version. Since this was just added in the previous release, I think just noting the break is sufficient. I've added a changelog entry.